pkg/rpc/response/result/invoke.go

```diff
@@ -69,6 +83,18 @@ func (r *Invoke) UnmarshalJSON(data []byte) error {
 			r.Stack = st
 		}
 	}
+	if aux.Transaction != nil {
+		bytes, err := hex.DecodeString(*aux.Transaction)
+		if err != nil {
+			return fmt.Errorf("failed to decode transaction bytes from hex: %w", err)
+		}
+		r.Transaction = new(transaction.Transaction)
```

Comment:
It's not possible to set the proper netmode right in UnmarshalJSON as far as we can't change the signature of this method. However, it's possible for RPC client to solve this problem, so I fixed this by filling netmode after JSON unmarshalling in `(c *Client) invokeSomething`. As a result, we get proper `Invoke` structure after `invokeSomething`. 

I don't see the other option of retrieving transaction netmode from the byte representation, so I'd rather drop this commit in case if proposed solution is not correct.

Reply:
Or we can just store unmarshalled bytestring in `Invoke.Transaction` field.

Reply:
Fixed by storing transaction bytes.Also added `(Invoke).GetTransaction(magic)` method to decode bytes into transaction.